internal/backup/csi_snapshotter.go

```diff
@@ -173,13 +174,24 @@ func (p *CSISnapshotter) Execute(item runtime.Unstructured, backup *velerov1api.
 			Namespace:     upd.Namespace,
 			Name:          upd.Name,
 		},
-		{
-			GroupResource: kuberesource.VolumeSnapshotContents,
-			Name:          snapshotContent.Name,
-		},
 	}
 
-	p.Log.Debug("Listing additional items to backup")
+	if util.IsVolumeSnapshotClassHasListerSecret(snapshotClass) {
```

Comment:
Should this block of logic move into its own backup item action, that runs for VolumeSnapshotClasses? That way any time we back up a VolumeSnapshotClass, we back up its related secret. Also keeps this plugin simpler.